sonar-plugin-api/src/main/java/org/sonar/api/batch/SensorContext.java

```diff
@@ -32,7 +32,9 @@
 
 /**
  * @since 1.10
+ * @deprecated since 5.6 use {@link org.sonar.api.batch.sensor.Sensor}
  */
+@Deprecated
 public interface SensorContext extends org.sonar.api.batch.sensor.SensorContext {
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Do not forget to remove this deprecated code someday. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3AS1133)